var/spack/repos/builtin/packages/relion/package.py

```diff
@@ -34,6 +34,10 @@ class Relion(CMakePackage):
     homepage = "http://http://www2.mrc-lmb.cam.ac.uk/relion"
     url      = "https://github.com/3dem/relion"
 
+    version('2.0.3', git='https://github.com/3dem/relion.git',
+                commit='50505945ec46c25ddebf7292492df7de331a2696')
+    version('v.2.1-beta-1', git='https://github.com/3dem/relion.git',
```

Comment:
no need for `v.` here, also put it above `2.0.3`, where you can add `preferred=true` if you want to keep it as default.